geomstats/geometry/special_euclidean.py

```diff
@@ -215,14 +214,13 @@ def belongs(self, point):
         Returns
         -------
         belongs : array-like, shape=[...,]
-            Boolean indicating whether point belongs to SE(3).
+            Boolean indicating whether point belongs to SE(2).
         """
         point_dim = point.shape[-1]
         point_ndim = point.ndim
         belongs = gs.logical_and(point_dim == self.dim, point_ndim < 3)
-
         belongs = gs.logical_and(
-            belongs, self.rotations.belongs(point[..., :self.n]))
+            belongs, self.rotations.belongs(point[..., :self.rotations.dim]))
```

Comment:
Does this work? self.n is different from self.rotations.dim

Reply:
Indeed, they only (?) coincide in the 3D case. For SE(2), the rotation is 1-dimensional, while n=2